src/main/java/com/java_essential/diledsys/myExperiments/loger/SimpleLog.java

```diff
@@ -0,0 +1,55 @@
+package com.java_essential.diledsys.myExperiments.loger;
+
+import java.io.*;
+import java.net.URLDecoder;
+import java.util.Calendar;
+
+public class SimpleLog {
+    private static FileOutputStream fis;
+
+    public static void log(String str) {
+        if (fis == null) getPath();
+        try {
+            fis.write(date().getBytes());
+            fis.write(str.getBytes());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static void getPath() {
+        File currentClass = null;
+        try {
+            currentClass = new File(URLDecoder.decode(SimpleLog.class
+                    .getProtectionDomain()
+                    .getCodeSource()
+                    .getLocation()
+                    .getPath(), "UTF-8"));
+        } catch (UnsupportedEncodingException e) {
+            e.printStackTrace();
+        }
+        new File(currentClass.getPath() + "\\log\\").mkdir();
+        try {
+            fis = new FileOutputStream(new File(currentClass.getPath() + "\\log\\" + "log.txt"), true);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void log(String str, String path) {
+        try {
+            fis = new FileOutputStream(new File(path + "log.txt"), true);
+            fis.write(date().getBytes());
+            fis.write(str.getBytes());
+        } catch (FileNotFoundException e) {
```

Comment:
Why at different catch?
FileNotFoundException its pat of IOException - it can be one catch.